src/theme/app/THEME.md

```diff
@@ -0,0 +1,20 @@
+## Adding dark theme for components
+
+1. To differentiate the colors between dark theme and light theme, we have already created a customisation system in which we can define style like colors, position, display etc for both theme using {props => props.theme.DarkTheme? darkThemeStyle: LightThemeStyle} 
+  
+2. It is not recommended to fill style sheet with css variables, if possible use only colors or styles that are already defined in themeStyles.js.
```

Comment:
This point is a bit unclear. You should probably phrase it as "It is recommended to use pre-existing styles or colors from the theme. Avoid adding your own CSS variables."